transport-classes-io_uring/src/main/java/io/netty/channel/uring/IoUringIoOps.java

```diff
@@ -349,4 +380,45 @@ public static IoUringIoOps newSend(
     public static IoUringIoOps newShutdown(int fd, int flags, int how, short data) {
         return new IoUringIoOps(Native.IORING_OP_SHUTDOWN, flags, (short) 0, fd, 0, 0, how, 0, data);
     }
+
+    /**
+     *
+     * Returns a new {@code OP_SPLICE} {@link IoUringIoOps}.
+     *
+     *  io_uring_prep_splice() - Either @fd_in or @fd_out must be a pipe.
+     *  - If @fd_in refers to a pipe, @off_in is ignored and must be set to -1.
+     *  - If @fd_in does not refer to a pipe and @off_in is -1, then @nbytes are read
+     *  from @fd_in starting from the file offset, which is incremented by the
+     *  number of bytes read.
+     *  - If @fd_in does not refer to a pipe and @off_in is not -1, then the starting
+     *  offset of @fd_in will be @off_in.
+     *  <p>
+     *  This splice operation can be used to implement sendfile by splicing to an
+     *  intermediate pipe first, then splice to the final destination.
+     *  In fact, the implementation of sendfile in kernel uses splice internally.
+     *  NOTE that even if fd_in or fd_out refers to a pipe, the splice operation
+     *  can still fail with EINVAL if one of the fd doesn't explicitly support splice
+     *  operation, e.g. reading from terminal is unsupported from kernel 5.7 to 5.11.
```

Comment:
Should this be formatted more javadoc-like? The bulleted list, for instance, will not render so well, and the `@` prefixed identifiers could be `{@code …}` bracketed.

Reply:
cleaned up